po/wesnoth-tutorial/hr.po

```diff
@@ -1754,7 +1754,7 @@ msgstr ""
 #, fuzzy
 #| msgid "female^Fighter"
 msgid "female^Warrior"
-msgstr "žena^Borac"
+msgstr "Borac"
```

Comment:
Not an issue for this PR, but it seems Hungarian translates both the L1 and the L2 as the same word.

Reply:
The lines before this indicates is fuzzy - the previous msgid was Fighter. Leaving this as-is.